src/views/registerForm.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Container, Alert, Form } from 'react-bootstrap';
+import useRegisterController from '../controllers/registerController.js';
+
+export default function RegisterForm() {
+  const [firstname, setFirstname] = useState('');
+  const [lastname, setLastname] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState('');
+  const { handleRegister, error } = useRegisterController();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    handleRegister({ firstname, lastname, username, password, email });
+  };
+
+  return (
+    <Container className='login-form-container'>
```

Comment:
Thinking about, since you are reusing `login-form-container` CSS class(and others below), let's rename them to be more generic, like `form-container`?